huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go

```diff
@@ -66,7 +72,7 @@ func ResourceFunctionGraphTrigger() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 				ValidateFunc: validation.StringInSlice([]string{
-					timingTrigger, obsTrigger, smnTrigger, disTrigger, kafkaTrigger,
+					timingTrigger, obsTrigger, smnTrigger, disTrigger, kafkaTrigger, apigTrigger, dedicatedApigTrigger,
```

Comment:
pls add new valid values APIG and DEDICATEDGATEWAY for `type` in the docs

Reply:
yeah, I forgot